pymc_marketing/mmm/models/components/saturation.py

```diff
@@ -0,0 +1,82 @@
+from pymc_marketing.mmm.models.components.base import Transformation
+from pymc_marketing.mmm.transformers import (
+    hill_saturation,
+    logistic_saturation,
+    michaelis_menten,
+    tanh_saturation,
+    tanh_saturation_baselined,
+)
+
+
+class SaturationTransformation(Transformation):
```

Comment:
Can you please add a doctor as in the `AdstockTransformation` above?

Reply:
```suggestion python
class SaturationTransformation(Transformation):
    """Subclass for all saturation transformations. 
    
    In order to use a custom saturation transformation, subclass and define: 
    
    - `function`: function to take x to contributions
    - `default_priors`: default distributions for each parameter in function
    
    By subclassing from this method, lift test integration will come for free!
    
    Examples
    ----------
    Make a non-saturating saturation transformation
    
    .. code-block:: python
        
        def infinite_returns(x, b): 
            return b * x
            
            
        class InfiniteReturns(SaturationTransformation): 
            function = infinite_returns
            
            default_priors = {"b": {"dist": "HalfNormal", "kwargs": {"sigma": 1}}}
    
    """
```